src/pages/trading-specification/components/_elements.tsx

```diff
@@ -12,7 +12,7 @@ import useRegion from 'components/hooks/use-region'
 import dl from 'images/svg/trading-specification/dl.svg'
 
 export const TableContainer = styled.div`
-    display: flex;
+    display: grid;
```

Comment:
Can we reach our needs with flex? With Atomic design we'll never use that at all